purchase_delivery_split_date/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2014-2016 Numérigraphe SARL
+# Copyright 2017 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Purchase Delivery Split Date",
+    "version": "12.0.1.0.0",
+    "summary": "Allows Purchase Order you confirm to generate one Incoming "
+               "Shipment for each expected date indicated in the Purchase "
+               "Order Lines",
+    "author": u"Numérigraphe, Eficent, Odoo Community Association (OCA)",
```

Comment:
`u` ??

Reply:
When i tried to generate the readme with ```oca-gen-addon-readme```, i had problems with the accent (see the author Numérigraphe). So i had to put that 'u' to say that string is in unicode.

Reply:
> When i tried to generate the readme with `oca-gen-addon-readme`, i had problems with the accent (see the author Numérigraphe). So i had to put that 'u' to say that string is in unicode.

@bjeficent IMO Change `Numérigraphe`  to `Numerigraphe` and try to generate the readme with `oca-gen-addon-readme`.

Reply:
Updates done.